src/ConstructiveSolidGeometry/PointsAndVectors.jl

```diff
@@ -21,7 +21,7 @@ end
 
 @inline (*)(p::CartesianPoint{T}, v::SVector{3,T}) where {T} = p .* v
 @inline (*)(vp::Vector{CartesianPoint{T}}, v::SVector{3,T}) where {T} = vp .* (v,)
-@inline (*)(vp::Vector{Vector{CartesianPoint{T}}}, v::SVector{3,T}) where {T} = vp .* (v,)
+@inline (*)(vvp::Vector{Vector{CartesianPoint{T}}}, v::SVector{3,T}) where {T} = vvp .* (v,)
```

Comment:
This, is actually a dangerous notation. Here, I would drop the `*`-method and would call `scale!` or `scale`

Reply:
By the way, `translate`, `rotate`, `translate!` and `rotate!` are also methods that are exported by Plots.jl.
When explicitly importing these functions from SSD, it gives a warning in either SSD or Plots.
For this, using `(+)` for translations and `(*)` for rotations would come in quite handy.